test/Microsoft.IdentityModel.JsonWebTokens.Tests/ActClaimTests/ActClaimSerializationTests.cs

```diff
@@ -0,0 +1,587 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Text.Json;
+using Microsoft.IdentityModel.TestUtils;
+using Microsoft.IdentityModel.Tokens;
+using Xunit;
+namespace Microsoft.IdentityModel.JsonWebTokens.Tests.ActClaimTests
+{
+    public class ActClaimSerializationTests
+    {
+        [Fact]
+        public void ActorTokenInClaimsDictionaryShouldBeProperlySerialized()
```

Comment:
I suggest adding underscores to the test names like in other tests. Something like:
- `ActorTokenInClaimsDictionary_CorrectlySerialized`
- `NestedActorToken_InClaimsDictionary_IsCorrectlySerialized`

See guidelines here: https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#follow-test-naming-standards